src/workers/cdc/primary.ts

```diff
@@ -624,6 +624,7 @@ const onSourceChange = async (
   logger: FastifyBaseLogger,
   data: ChangeMessage<Source>,
 ) => {
+  console.log('source operation: ', data.payload.op);
```

Comment:
probably wont to use `logger`?

Reply:
```suggestion
logger.info(
{ op, before, after },
'source change',
);
```

do everything in single log?

Reply:
was mainly afraid it could spool to many on creation, but we can try for a short term anyway.